recipes/leftbar/style/leftbar_row.less

```diff
@@ -272,15 +272,10 @@
     white-space: nowrap;
   }
 
-  &__meta-context {
-    //
-  }
 
-  &__meta-custom:not(:empty) {
-    &:deep(.dt-leftbar-row__meta-context ~ &:before) {
-      content: ' • ';
-      color: var(--dt-theme-sidebar-status-color-foreground);
-    }
```

Comment:
Just remove the `&:deep` and it'd be working correctly.
I added that by mistake.

```css
&__meta-custom:not(:empty) {
  .dt-leftbar-row__meta-context ~ &:before {
    content: ' • ';
    color: var(--dt-theme-sidebar-status-color-foreground);
  }
}
```

Reply:
Yeah, good catch on that tiny detail — I totally forgot about that. The `•` should indeed appear when `custom` follow `context` inline.

Reply:
Updated

Reply:
I recognize this is kind of a CSS workaround, such that the logic is in the CSS rather than in the Vue. 

Would it be cleaner to do a conditional, which can just insert it in between? Like say...

```html
<div class="dt-leftbar-row__status">
  <span class="dt-leftbar-row__meta-context">{presenceText}</span>
  <span aria-hidden="true">&bull;</span>
  <span class="dt-leftbar-row__meta-custom">{userstatus}</span>
</div>
```